bigbluebutton-html5/imports/ui/Types/user.ts

```diff
@@ -109,3 +109,18 @@ export interface User {
   sessionCurrent: sessionCurrent;
   livekit?: Livekit;
 }
+
+export interface UserBasicInfo {
+  userId: string;
+  extId: string;
+  name: string;
+  isModerator: boolean;
+  role: string;
+  currentlyInMeeting: boolean;
```

Comment:
`currentlyInMeeting` is not necessary as it is always be `true`. It is a condition to show the user, see:

https://github.com/bigbluebutton/bigbluebutton/blob/0892f73d73ba2cbe324d9d7b7f8988e97c40284b/bbb-graphql-server/bbb_schema.sql#L434